config/src/test/resources/org/springframework/security/config/annotation/web/configurers/oauth2/server/resource/OAuth2ResourceServerConfigurerTests-WrongAlgorithm.token

```diff
@@ -0,0 +1 @@
+eyJhbGciOiJSUzUxMiJ9.eyJleHAiOjQ2ODczNDQ2NzN9.hvVUW_xwUXd7nGm27E5tLTZ21x64YjP0o-TMW6t_bOkfG1Vp1AMEX8fXvSqeG0vK8TWiB2_keOGtH-eFmAGBEYXq1o1zj1BgMHeaZAVio9n-77DkTzQ7CiOF5M1M7B_Ng4K8ra4DpieZZXVjHTWsuOiU1hWoI1tIna8VucAxZln-oh7PkrYmgwFTlsL2Z9aZZYN_X7ECyRQDf3lRrLwr4Go_XpJ5i9F-GT5LvUYa42uggGjvq_frfb0t5wcmPgjtqiE6l2mnrYFjjKTq1nQRYrJ5wFWOHUTRxNsGS8PwrNxzh6JW1ZZTS0n_JIOvSh__w0WAB241QLoKBx4AETMLQA
```

Comment:
I've also been rethinking our tokens being in external files. I think it might be better to just have a static constant in the test file. This would certainly reduce IO for tests and speed them up. It would also be less jumping around to figure out what is being done.

If we decide to do this, it would probably make sense to keep things as is in this PR and create a separate ticket to change it.

Thoughts?

Reply:
I think the two competing motives are first have the tests be quick and accurate and second to show typical usage to individuals inspecting the tests for tips. The later motive is why the xml config tests have their configs in separate xml files.

I don't see the second benefit with the tokens being in separate files, so yes, I think it would clean up the tests here to not have the separate files for the tokens, but instead use static constants.

Reply:
Thanks can you create a ticket @jzheaux ?